pup-0006.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Commit Bit Assignment
+author: Ina Panova
+created: 15-08-2018
+status: Approved
+version: 1
+---
+
+## Summary
+
+For pulpcore and its related tools, there is no written process to describe giving the commit bit
+to a contributor. The existing committers want to openly document the process.
+
+
+## Motivation
+
+Comitter - is an individual who is able to modify the source code and then commit the changes into
+a Pulp's [Github](https://github.com/pulp/) organization repository.
+To have "a commit bit" means that one is permitted to commit those changes. 
+
+Historically the commit bit was given on day 0 when hired on the core team at Red Hat.  In Oct 2017
+the existing committers decided to stop that practice and instead document an open process.
+Engineers hired on the Pulp team since Oct '17 have not received commit bit.
+
+We have not yet documented an open process of which to become a comitter.
+The process outlined in this document is inspired by processes in Chromium projects and Apache
+Software Foundation.
+
+
+## Scope
+
+This policy is intended as the default for all software in the Pulp [Github](https://github.com/pulp/)
+organization unless explicitly stated otherwise.
+
+
+## Criteria
+
+Becomming a comitter.
```

Comment:
Can this get moved up and replace `## Criteria` so it would be `## Becoming a committer`.